src/util/BaseLoginRouter.js

```diff
@@ -22,8 +22,10 @@ define([
   'views/shared/SecurityBeacon',
   'views/shared/AuthContainer',
   'models/AppState',
+  'models/WidgetState',
   './ColorsUtil',
   './RouterUtil',
+  './RoutingUtil',
```

Comment:
This is a pretty subtle/confusing name difference, `RouterUtil` vs `RoutingUtil`. Maybe something more descriptive? Could be `IdxRouterUtil` `IdxPipelineUtil` or even `RouterV2Util`

Reply:
Agreed. I was hoping to get some suggestion for the name here TBH. I want to stay away form having Idx in our file names since it may not make sense in the long run and its mostly an internal term. How about `NavigationUtil` , `viewLoader`  or `responseParser` since it basically just parses the status in the response and loads a view ?